x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -70,6 +70,7 @@ const FormattedFieldValueComponent: React.FC<{
   eventId: string;
   isAggregatable?: boolean;
   isObjectArray?: boolean;
+  isUnifiedDataTable?: boolean;
```

Comment:
Reminder @logeekal , need to check the need for this.

Reply:
I think it may have to do with a couple things. In the current timeline table we have this tooltip, essentially showing the date in multiple formats. This would be a deprecation, but not sure how many people use it

<img width="390" alt="Screenshot 2024-03-08 at 1 10 26 PM" src="https://github.com/elastic/kibana/assets/17211684/9d8a045c-bc53-4633-aa4b-30a3ddd16f90">

There's also the check for `dragAndDrop` but will comment there about that

Reply:
Okay... It make sense now and I have removed most of the logic from `field_formatters_udt`

 See a141c2b1c6ca7f9dce37bbed0e0001793c808482.